### PR TITLE
Handle recent versions of Ibis

### PR DESCRIPTION
**Pull request by philippjfr**
Adds support for ibis > 2.0

### CODE REVIEW

File: holoviews/core/data/ibis.py [hunk 1]
```diff
@@ -34,8 +34,11 @@ def has_rowid(cls):
 
     @classmethod
     def is_rowid_zero_indexed(cls, data):
-        from ibis.client import find_backends, validate_backends
-        (backend,) = validate_backends(list(find_backends(data)))
+        try:
+            from ibis.client import find_backends, validate_backends
+            (backend,) = validate_backends(list(find_backends(data)))
+        except Exception:
+            backend = data._find_backend()
```
Comment (jlstevens) [inline holoviews/core/data/ibis.py:41]: That wasn't so bad after all!

**Comment by jlstevens** [comment]
Tests are going green. Merging.

**Comment by maximlt** [comment]
Thanks @philippjfr ! The tests that ran in this PR were using ibis < 2 so not really testing the change, I'm hopeful I can get the CI to actually test ibis >= 2.

**Comment by philippjfr** [comment]
>Thanks @philippjfr ! The tests that ran in this PR were using ibis < 2 so not really testing the change, I'm hopeful I can get the CI to actually test ibis >= 2.

That would be great!

**Pull request merged**